tests/tracing/export/test_inference_table_exporter.py

```diff
@@ -13,7 +11,7 @@
 from mlflow.tracing.trace_manager import InMemoryTraceManager
 from mlflow.tracing.utils import encode_span_id, encode_trace_id
 
-from tests.tracing.helper import create_mock_otel_span, create_test_trace_info
+from tests.tracing.helper import _dump_attributes, create_mock_otel_span, create_test_trace_info
```

Comment:
Implementing `Trace.from_dict()` would be super helpful here (https://github.com/mlflow/mlflow/pull/12029/files#r1605367926) because it will allow us to add a test case that:

1. Creates an MLflow Trace
2. Exports the trace to a table
3. Calls MLflow Trace.from_dict() on the table and verifies that the obtained trace is equivalent to the original trace from (1)